app/models/user.rb

```diff
@@ -112,7 +112,14 @@ def student
   # This field is used to skip validations when creating
   # a preliminary user, e.g. when adding a non existant person
   # to a team using the github handle.
-  attr_accessor :github_import
+  attr_reader :github_import
+
+  # This informs devise that this user does not
+  # need a confirmation notification for now
+  def github_import=(import)
+    @github_import = import
+    skip_confirmation_notification! if @github_import
```

Comment:
One very minor thing: I would swap the two lines so that it returns `@github_import`; one would expect an assignment to return its value.